clients/src/test/java/org/apache/kafka/clients/consumer/ConsumerConfigTest.java

```diff
@@ -193,6 +193,17 @@ public void testRemoteAssignorConfig() {
         assertEquals(remoteAssignorName, consumerConfig.getString(ConsumerConfig.GROUP_REMOTE_ASSIGNOR_CONFIG));
     }
 
+    @Test
+    public void testRemoteAssignorWithClassicGroupProtocol() {
+        String remoteAssignorName = "SomeAssignor";
+        final Map<String, Object> configs = new HashMap<>();
+        configs.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, keyDeserializerClass);
+        configs.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, valueDeserializerClass);
+        configs.put(ConsumerConfig.GROUP_REMOTE_ASSIGNOR_CONFIG, remoteAssignorName);
+        InvalidConfigurationException exception = assertThrows(InvalidConfigurationException.class, () -> new ConsumerConfig(configs));
```

Comment:
Per the above, this would need to change too:

```suggestion
        ConfigException exception = assertThrows(ConfigException, () -> new ConsumerConfig(configs));
```